deploy/tasks/buildah-oci-ta.yaml

```diff
@@ -0,0 +1,744 @@
+        image
+      type: array
+      default: [ ]
+    - name: PREFETCH_INPUT
+      description: In case it is not empty, the prefetched content should
+        be made available to the build.
+      type: string
+      default: ""
+    - name: SKIP_UNUSED_STAGES
+      description: Whether to skip stages in Containerfile that seem unused
+        by subsequent stages
+      type: string
+      default: "true"
+    - name: SOURCE_ARTIFACT
+      description: The Trusted Artifact URI pointing to the artifact with
+        the application source code.
+      type: string
+    - name: SQUASH
+      description: Squash all new and previous layers added as a part of this
+        build, as per --squash
+      type: string
+      default: "false"
+    - name: STORAGE_DRIVER
+      description: Storage driver to configure for buildah
+      type: string
+      default: vfs
+    - name: TARGET_STAGE
+      description: Target stage in Dockerfile to build. If not specified,
+        the Dockerfile is processed entirely to (and including) its last stage.
+      type: string
+      default: ""
+    - name: TLSVERIFY
+      description: Verify the TLS on the registry endpoint (for push/pull
+        to a non-TLS registry)
+      type: string
+      default: "true"
+    - name: YUM_REPOS_D_FETCHED
+      description: Path in source workspace where dynamically-fetched repos
+        are present
+      default: fetched.repos.d
+    - name: YUM_REPOS_D_SRC
+      description: Path in the git repository in which yum repository files
+        are stored
+      default: repos.d
+    - name: YUM_REPOS_D_TARGET
+      description: Target path on the container in which yum repository files
+        should be made available
+      default: /etc/yum.repos.d
+    - name: caTrustConfigMapKey
+      description: The name of the key in the ConfigMap that contains the
+        CA bundle data.
+      type: string
+      default: ca-bundle.crt
+    - name: caTrustConfigMapName
+      description: The name of the ConfigMap to read CA bundle data from.
+      type: string
+      default: trusted-ca
+    - name: BUILD_IMAGE
+      description: The buildah image to use.
+      type: string
+      default: quay.io/konflux-ci/buildah-task:latest@sha256:5cbd487022fb7ac476cbfdea25513b810f7e343ec48f89dc6a4e8c3c39fa37a2
+    - name: DOMAIN_PROXY
+      description: Determines if domain proxy will be used when hermetic mode is enabled.
+      type: string
+      default: "false"
+    - name: BYTE_BUFFER_SIZE
+      description: The byte buffer size to use for the domain proxy.
+      type: string
+      default: 32768
+    - name: PROXY_TARGET_WHITELIST
+      description: Comma separated whitelist of target hosts for the domain proxy.
+      type: string
+    - name: INTERNAL_PROXY_ADDRESS
+      description: Host and port of proxy used internally by the domain proxy.
+      type: string
+    - name: INTERNAL_PROXY_USER
```

Comment:
Is this an up-to-date copy of buildah-oci-ta? I'm not sure if the copy in gitlab has any changes? Can you summarise the changes (somewhere?!?) ?

Reply:
It is an up to date copy. I have no idea what the one in gitlab has.

![Screenshot From 2024-11-29 11-23-20](https://github.com/user-attachments/assets/bf64d6ef-7758-45a2-88ae-9cb5863181dd)
![Screenshot From 2024-11-29 11-23-07](https://github.com/user-attachments/assets/8374261d-0be4-4e72-a743-ac542834cc32)
![Screenshot From 2024-11-29 11-22-57](https://github.com/user-attachments/assets/d4413b31-034a-451a-bc75-d1512da6ba11)
![Screenshot From 2024-11-29 11-22-49](https://github.com/user-attachments/assets/38bc4068-4ea5-4ae2-a9cd-d48460f8d37f)